templates/search/catalogue_search.html

```diff
@@ -31,48 +43,63 @@
             {% include './blocks/visualisation_menu.html' %}
         {% endif %}
 
-        {% if buckets.current.result_count > 0 %}
+        {# Map view is displayed without results and filters #}
+        {% if form.views.value == views.MAP.value %}
             <div class="tna-container">
                 <div class="tna-column tna-column--full">
-                    {% include './blocks/search_sort_and_view_options.html' %}
+                    {% include './blocks/search_results_map.html' %}
```

Comment:
Why are the div tna-container, tna-column tna-column--full - outside of block html - search_results_map.html? Is there any issue to include then inside the block html?

Reply:
I would suggest leaving them outside like this. That means that you could then include your block in whatever layout is needed, rather than the block defining the layout.

Reply:
Thanks for that @ahosgood , came to know in a discussion that its used this way in other places as well. 
On another note, lesser number of `div`'s make it easier to navigate to the what we're looking for. Is there any limit or practice for nesting div's?